src/app/containers/StakePage/components/HistoryEventsTable.tsx

```diff
@@ -299,7 +237,7 @@ const HistoryTable: React.FC<History> = ({ items }) => {
       {items.map((item, index) => {
         return (
           <HisoryTableAsset
-            key={item.transactionHash}
+            key={item.txHash + item.newBalance}
```

Comment:
`txHash` should be unique so I don't think you need to interpolate the `key`, do you?

Reply:
yes, they should be unique, but we have duplicated sometimes, need to check with @BetsyBraddock why is so
https://prnt.sc/1apo2ke

Reply:
It's weird but the reason can be we can have same tx hash for different action. But it's better to check with Betsy so we know we have valid data.

Reply:
checked with Betsy, its a bug, will be fixed on backed side today or tomorrow, so then i'll fix it here